.github/pull_request_template.md

```diff
@@ -1,15 +1,8 @@
----
-name: Pull request
-about: Create a pull request
-title: ''
-labels: ''
-assignees: ''
-
----
 ## General
 Please consider the infos from [CONTRIBUTING.md](CONTRIBUTING.md) before you open a pull request.
```

Comment:
What do you think about that "consider the infos" here? "infos" looks like an interesting word to me.
```suggestion
Please consider the guidelines from [CONTRIBUTING.md](CONTRIBUTING.md) before you open a pull request.
```